distributions/openhab/src/main/resources/runtime/etc/jetty.xml

```diff
@@ -105,35 +105,6 @@
 		<Set name="EndpointIdentificationAlgorithm"></Set>
 		<Set name="NeedClientAuth"><SystemProperty name="jetty.ssl.needClientAuth" default="false" /></Set>
 		<Set name="WantClientAuth"><SystemProperty name="jetty.ssl.wantClientAuth" default="false" /></Set>
-		<Set name="ExcludeCipherSuites">
```

Comment:
@wborn I assume this was removed because the newer Jetty excludes those by default?

Reply:
Hrmpf, maybe I should have read your commit message in more depth ;-)

> Jetty nowadays already excludes these weak cipher suites and protocols by default.

👍

Reply:
That's right! :-) It cleans up the file nicely and we'll be automatically in sync when in the future Jetty warns for (and excludes) more weak ciphers and protocols.